sdk/python/test/e2e/test_e2e_mxjob.py

```diff
@@ -20,92 +20,87 @@
+        ],
+        ports=[V1ContainerPort(container_port=9991, name="mxjob-port")],
     )
 
     server_container = V1Container(
-        name="mxnet",
+        name=CONTAINER_NAME,
         image="docker.io/johnugeorge/mxnet:1.9.1_cpu_py3",
-        ports=[V1ContainerPort(container_port=9991, name="mxjob-port")]
+        ports=[V1ContainerPort(container_port=9991, name="mxjob-port")],
     )
 
     scheduler_container = V1Container(
-        name="mxnet",
+        name=CONTAINER_NAME,
         image="docker.io/johnugeorge/mxnet:1.9.1_cpu_py3",
-        ports=[V1ContainerPort(container_port=9991, name="mxjob-port")]
+        ports=[V1ContainerPort(container_port=9991, name="mxjob-port")],
     )
 
     worker = V1ReplicaSpec(
         replicas=1,
         restart_policy="Never",
-        template=V1PodTemplateSpec(
-            spec=V1PodSpec(
-                containers=[worker_container]
-            )
-        )
+        template=V1PodTemplateSpec(spec=V1PodSpec(containers=[worker_container])),
     )
 
     server = V1ReplicaSpec(
         replicas=1,
         restart_policy="Never",
-        template=V1PodTemplateSpec(
-            spec=V1PodSpec(
-                containers=[server_container]
-            )
-        )
+        template=V1PodTemplateSpec(spec=V1PodSpec(containers=[server_container])),
     )
 
     scheduler = V1ReplicaSpec(
         replicas=1,
         restart_policy="Never",
-        template=V1PodTemplateSpec(
-            spec=V1PodSpec(
-                containers=[scheduler_container]
-            )
-        )
+        template=V1PodTemplateSpec(spec=V1PodSpec(containers=[scheduler_container])),
     )
 
     mxjob = KubeflowOrgV1MXJob(
         api_version="kubeflow.org/v1",
         kind="MXJob",
-        metadata=V1ObjectMeta(name="mxjob-mnist-ci-test", namespace=SDK_TEST_NAMESPACE),
+        metadata=V1ObjectMeta(name=JOB_NAME, namespace=SDK_TEST_NAMESPACE),
         spec=KubeflowOrgV1MXJobSpec(
             job_mode="MXTrain",
-            run_policy=V1RunPolicy(
-                clean_pod_policy="None",
-            ),
-            mx_replica_specs={"Scheduler": scheduler,
-                                "Server": server,
-                                   "Worker": worker}
-        )
+            run_policy=V1RunPolicy(clean_pod_policy="None",),
+            mx_replica_specs={
+                "Scheduler": scheduler,
+                "Server": server,
+                "Worker": worker,
+            },
+        ),
     )
 
-    MX_CLIENT.create(mxjob)
+    TRAINING_CLIENT.create_mxjob(mxjob, SDK_TEST_NAMESPACE)
 
-    MX_CLIENT.wait_for_job("mxjob-mnist-ci-test", namespace=SDK_TEST_NAMESPACE)
-    if not MX_CLIENT.is_job_succeeded("mxjob-mnist-ci-test",
-                                           namespace=SDK_TEST_NAMESPACE):
-        raise RuntimeError("The MXJob is not succeeded.")
+    TRAINING_CLIENT.wait_for_job_conditions(
+        JOB_NAME, SDK_TEST_NAMESPACE, constants.MXJOB_KIND
+    )
```

Comment:
If possible, can you add few more extra validations with other SDK functions as well  (which are missing now) like get_job_pod_names, list_mxjobs, is_job_succeeded, is_job_created, get_job_conditions etc

Reply:
@johnugeorge Sure, I'll add them.